### ISSUE TITLE
Better screen reader support

### ISSUE THREAD

**Issue by francipvb** [flutter/devtools#5795]
Hello,

Some time ago I started to develop flutter applications. Unfortunately I am a visually impaired user and I cannot use the devtools because it is not screen-reader compatible.

I often find myself putting a breakpoint and performing an hot reload to see variables and widget state in the debugger, as the devtools are not accessible to screen readers.
